benchmarks/bench_result_add.py

```diff
@@ -0,0 +1,101 @@
+import time
+from qcfractal.interface.models.records import ResultRecord
+from qcfractal.interface.models import KeywordSet
+import qcfractal
+import qcfractal.interface as ptl
+import numpy as np
+import qcelemental as qcel
+
+print("Building and clearing the database...\n")
+db_name = "molecule_tests"
+storage = qcfractal.storage_socket_factory(f"postgresql://localhost:5432/{db_name}")
+storage._delete_DB_data(db_name)
+
+run_tests = True
+mol_trials = [1, 5, 10, 25, 50, 100, 500, 1000]
+# _size = 20
+
+COUNTER_MOL = 0
+
+
+def build_unique_mol():
+    global COUNTER_MOL
+    mol = qcel.models.Molecule(symbols=["He", "He"], geometry=np.random.rand(2, 3) + COUNTER_MOL, validated=True)
+    COUNTER_MOL += 1
+    return mol
+
+
+def create_unique_result():
+    mol = build_unique_mol()
+    ret = storage.add_molecules([mol])["data"]
+    result = ResultRecord(version='1', driver='energy', program='games', molecule=ret[0],
+                          method='test', basis='6-31g')
+
+    return result
+
+mol1 = build_unique_mol()
+ret_mol = storage.add_molecules([mol1])["data"]
+result1 = ResultRecord(version='1', driver='energy',program='games',molecule=ret_mol[0], method='test', basis="")
+result2 = ResultRecord(version='1', driver='energy',program='games',molecule=ret_mol[0], method='test', basis=None)
+ret = storage.add_results([result1])["meta"]
+ret = storage.add_results([result2])["meta"]
```

Comment:
What are you hoping to test here? Please add an assert statement.

Reply:
Deleted this, it was just a sanity test for myself.